src/osd/OSD.cc

```diff
@@ -12,14 +12,19 @@
  * Foundation.  See file COPYING.
  *
  */
-#include "acconfig.h"
```

Comment:
Is there a reason you reordered these include orders? There are some annoyingly delicate dependencies and the cross-platform stuff won't get picked up in our own testing.

Reply:
Hm, I didn't know about the cross-platform issue! I have to say it's almost become reflexive. Sage has asked before that headers be tidied up "while we're in there", but perhaps acconfig wasn't on the menu. I've moved it back to the top. I can also revert all the re-ordering if that's the best thing to do.

Reply:
Well, I'm not saying there definitely is a problem; like I said the issues are fragile. And while our config header is a good guess for which one matters, it'll really be anything that inherits any kind of definition where we override or conflict with the standard ones. (We see the same thing with our own definition of assert, for instance, although that one's markedly less fragile.)
But in general I get queasy around seeing headers rearranged; the cosmetics aren't worth it in a system with as cross-unit dependencies and invariants as Ceph has.